projects/core/src/global-message/models/global-message.model.ts

```diff
@@ -10,4 +10,5 @@ export enum GlobalMessageType {
 export interface GlobalMessage {
   text: Translatable;
   type: GlobalMessageType;
+  duration?: number;
```

Comment:
For consistency, what do you think aligning the name of this property with the name used in the global message config - `timeout`?

Reply:
I changed the name of the property